include/private/rdysweimpl.h

```diff
@@ -11,11 +11,20 @@ typedef struct {
   PetscReal *u, *v;        // diagnostic variables
 } RiemannDataSWE;
 
+typedef struct {
+  PetscInt   N;        // number of data values
+  PetscReal *cn, *sn;  // cosine and sine value of a edge
```

Comment:
Perhaps "x and y components of edge normal"?

Reply:
We can scoop this up in the refactor.

Reply:
I think that's also a bit ambiguously worded (is it the edge or the normal to the edge?). Not critical now, but if we move to a curvilinear coordinate system, we'll want to be more precise about what quantity we're storing.